src/mono/wasm/testassets/WasmBasicTestApp/App/wwwroot/main.js

```diff
@@ -81,7 +81,7 @@ switch (testCase) {
     case "InterpPgoTest":
         dotnet
             .withConsoleForwarding()
-            .withRuntimeOptions(['--interp-pgo-logging'])
+            .withRuntimeOptions(['--interp-pgo-logging', '--interp-pgo-recording'])
```

Comment:
Should we wrap `--interp-pgo-recording` into `withInterpreterPgo`?
They are needed both for pgo to be working, am I right?

Reply:
Recording is separate from using the data, so it might make sense for them to be separate options. But it's a good idea, I'll consider it.